
<launch>

	<machine name="local_alt" address="localhost" default="true" />
	
	<node name="RatSLAMPoseCells" pkg="ratslam_ros" type="ratslam_pc" args="$(find ratslam_ros)/tmp_config.txt _image_transport:=compressed" cwd="node" required="true">
		<remap from="/irat_red/odom" to="/odom"/>
	</node>
	
</launch>
